README.md

```diff
@@ -214,7 +214,7 @@ human and sometimes make mistakes. To prevent accidental breaking changes, you
 can also pin to a specific version:
 
 ```yaml
-- uses: 'google-github-actions/setup-gcloud@v0.1.1'
```

Comment:
This was intentional. We won't be updating the README to point to the latest point release all the time.

Reply:
thank you for reviewing.
Oh, I see. Sorry for my misunderstanding.
Let me fix the title of PR.